services/clojars/clojars-downloads.service.js

```diff
@@ -44,16 +44,15 @@ module.exports = class Clojars extends BaseJsonService {
   static get route() {
     return {
       base: 'clojars/dt',
-      format: '(.+)',
-      capture: ['clojar'],
+      pattern: ':clojar',
```

Comment:
Should the pattern be `:clojar(.+)` here to support [group/artifact targets](https://github.com/clojars/clojars-web/wiki/Data#artifact-httpsclojarsorgapiartifactsgroup_nameartifact_name)?

I believe it's possible that folks are using the badge with something like `/clojars/dt/cljsjs/react.svg`:

https://img.shields.io/clojars/dt/cljsjs/react.svg
![https://img.shields.io/clojars/dt/cljsjs/react.svg](https://img.shields.io/clojars/dt/cljsjs/react.svg) 

Which wouldn't work with this pattern:
https://shields-staging-pr-2633.herokuapp.com/clojars/dt/cljsjs/react.svg

Reply:
Gotcha. Yea, I think actually `':clojar+'` is the succinct way of saying there can be a slash.